docs/query-metadata-style-guide.md

```diff
@@ -157,6 +157,15 @@ If your query is a security query, use one or more `@tags` to associate it with
 
 When you tag a query like this, the associated CWE pages from [MITRE.org](http://cwe.mitre.org/index.html) will automatically appear in the reference section of its associated qhelp file.
 
+#### Metric/summary `@tags`
+
+Metrics and other pieces of summary information will be output in the SARIF format according to the [specification](https://docs.google.com/document/d/1V3zpkj0OGh8GEUVwACRx7fiafE5zklujAftZaYUyf9s/edit#heading=h.d6vfrrmrajbb) agreed with the Code Scanning team.
```

Comment:
This prompts me to login, so I guess you should probably replace this link with something that is "more" public.

Reply:
Thanks for the pointers. The internal links aren't suitable for inclusion here. Once `summary` queries are more widespread I hope we will provide external documentation for integration and then we could link to that here.